katdal/datasources.py

```diff
@@ -632,7 +633,7 @@ def from_url(cls, url, chunk_store='auto', upgrade_flags=True, **kwargs):
         db = int(kwargs.pop('db', '0'))
         if url_parts.scheme == 'file':
             # RDB dump file
-            telstate = katsdptelstate.TelescopeState(katsdptelstate.memory.MemoryBackend())
+            telstate = katsdptelstate.TelescopeState()
```

Comment:
Actually I see that the change in telstate to make this the default didn't make it into the 0.7 release. I'd suggest reverting this change until we have a new telstate release (and then make katdal depend on it), since there will probably be users using PyPI katsdptelstate and git master katdal.